src/main/java/bio/terra/cli/command/resource/addref/GcsObject.java

```diff
@@ -41,10 +52,29 @@ protected void execute() {
     // build the resource object to add
     CreateResourceParams.Builder createResourceParams =
         referencedResourceCreationOptions.populateMetadataFields();
+
+    // parsing the path as bucket name and object name
+    if (gcsPath != null) {
+      if (bucketName != null || objectName != null) {
+        throw new UserActionableException("Specify only one path to add reference.");
```

Comment:
Can you make the error more specific?  Like "Specify either --gcs-path or both --bucket-name and --object-name"